domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -104,7 +104,7 @@ class ExplorationProgressController @Inject constructor(
   private val translationController: TranslationController,
   private val dataProviders: DataProviders,
   private val loggingIdentifierController: LoggingIdentifierController,
-  private val profileManagementController: ProfileManagementController,
+  val profileManagementController: ProfileManagementController,
```

Comment:
Why is this been changed to public? That shouldn't be needed.

Reply:
done